docs/source/definition-of-done.md

```diff
@@ -0,0 +1,12 @@
+# Definiton of done
+
+ * [ ] The acceptance criteria described in the Task is met
+ * [ ] The code is approved and merged into the repository(ies)
+ * [ ] The documentation is updated
+   * Official landing page: [](https://leapp.readthedocs.io/en/latest/contributing.html)
+   * Developers documentation: https://leapp.readthedocs.io/en/latest/
```

Comment:
```suggestion
   * Upstream documentation: https://leapp.readthedocs.io/en/latest/
```

Reply:
waiting for a decision regarding existency of definition of done